### ISSUE TITLE
Viewing text files by default

### ISSUE THREAD

**Issue by m-schmitt** [QL-Win/QuickLook#807]
QuickLook recognizes certain specific extensions (e.g. .txt) as being a text file. The problem is that there are a large number of files that are simple text files, but have different extensions, or no extension at all.
 
On macOS there is a QuickLook plugin [QuicklookStephen](https://github.com/whomwah/qlstephen) that solves this. This plugin essentially makes QuickLook try to open any file that is not otherwise recognized as a text file.

I suggest a similar plugin for QuickLook for Windows.

Note: QuicklookStephen's logic is described [here](https://github.com/whomwah/qlstephen): it previews everything as text if:

- It is associated with the UTI public.data
- It is not blacklisted
- And there is not a different, more type-specific Quick Look plugin registered for the file type

**Comment by xupefei** [comment]
> QuickLook recognizes certain specific extensions (e.g. .txt) as being a text file. The problem is that there are a large number of files that are simple text files, but have different extensions, or no extension at all.

This is not true. QuickLook checks first 16KB of a given file for two adjacent 0x00. If no then the file is treated as text: https://github.com/QL-Win/QuickLook/blob/master/QuickLook.Plugin/QuickLook.Plugin.TextViewer/Plugin.cs#L77.

Did you encounter a problem?

**Comment by m-schmitt** [comment]
Ah, I tried it on some random files when I first installed it, and didn't recognize as text, but now it does. Perhaps this was because the website promoting QuickLook was linking to the Windows Store version, and I installed it before realizing there was a stand-alone, more up-to-date version here.  (Which I'd rather use anyway; I only use Windows Store apps as a last resort.)

Now I'm seeing the text files previewed as text. Yay.

(Also, this utility is great: I've long wished for a Windows version of Quick Look.)
